nodejs/examples/elasticsearch.js

```diff
@@ -1,62 +1,63 @@
-const elasticsearch = require('elasticsearch');
+const elasticsearch = require("elasticsearch");
 const config = require("platformsh-config").config();
 
-exports.usageExample = async function() {
+exports.usageExample = async function () {
+    const credentials = config.credentials("elasticsearch");
 
-    const credentials = config.credentials('elasticsearch');
-
-    var client = new elasticsearch.Client({
+    const client = new elasticsearch.Client({
         host: `${credentials.host}:${credentials.port}`,
     });
 
-
-    let index = 'my_index';
-    let type = 'People';
+    const index = "my_index";
+    const type = "People";
 
     // Index a few document.
-    let names = ['Ada Lovelace', 'Alonzo Church', 'Barbara Liskov'];
+    const names = ["Ada Lovelace", "Alonzo Church", "Barbara Liskov"];
 
-    let message = {
+    const message = {
         refresh: "wait_for",
-        body: []
+        body: names.flatMap((name) => [
+            { index: { _index: index, _type: type } },
+            { name },
+        ]),
     };
-    names.forEach((name) => {
-        message.body.push({index: {_index: index, _type: type}});
-        message.body.push({name: name});
-    });
+
     await client.bulk(message);
 
     // Search for documents.
     const response = await client.search({
-        index: index,
-        q: 'name:Barbara Liskov'
+        index,
+        q: "name:Barbara Liskov",
```

Comment:
Why no `index` key  name?  Isn't it required in an object?

Reply:
when the key and the variable name holding the value are the same, you can just do `{ someKey }`. That's equivalent to `{ someKey: someKey }`

Reply:
http://es6-features.org/#PropertyShorthand